### PR TITLE
Update project website

### LINKED ISSUE #28: Update project website
- [ ] Update project website user guide to reflect tables as shown in /docs/UserGuide.md
- /docs/UserGuide.md `add` and `list` sections' tables show correctly
- but [project website user guide](https://ay2324s1-cs2103-t16-1.github.io/tp/UserGuide.html) `add` and `list` sections' tables has wrong formatting

- [ ] Update project website homepage to show our **new** README.md
- currently displaying AddressBook Level-3's details

### PR DESCRIPTION
**Pull request by kenvynKwek**
Updated:
- UserGuide `add` feature section's 'Expected outputs' table format to display correctly
- 'README.md' CI Status badge
- 'index.md' to change project website's homepage to reflect 'README.md' and display ManageHR's details

This would close #28

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,19 +1,50 @@
...
+
+Johnny Sins, a HR staff
+> 5/5 :star: "Wow ! I wish I discovered an application like this sooner"
+
+Ricardo Milos, a HR intern
+> 4.5/5 :star: "Revolutionary technology, good stuff"
+
+## Links
+
+If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
```
Comment (SungMatt) [inline docs/index.md]: Should be ManageHR here

File: docs/index.md [hunk 2]
```diff
@@ -1,19 +1,50 @@
...
 
-* If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
-* If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
+If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
```
Comment (SungMatt) [inline docs/index.md]: Same for this line! ManageHR

**Review by SungMatt — CHANGES_REQUESTED** [review]
Few things to update, otherwise looks good!

File: docs/index.md (hunk 1 above)
Comment (kenvynKwek) [reply docs/index.md]: Thanks for catching this !

File: docs/index.md (hunk 2 above)
Comment (kenvynKwek) [reply docs/index.md]: Thanks for catching this too !

**Comment by kenvynKwek** [comment]
Updated lines 42 and 44 that mentioned 'AddressBook' to 'ManageHR' as per review comments

**Review by SungMatt — APPROVED** [review]
LGTM! Thanks for the changes!

**Comment by SungMatt** [comment]
Remember to tag this with the appropriate milestone!

**Pull request merged**